modules/javafx.graphics/src/main/java/javafx/css/PseudoClass.java

```diff
@@ -81,18 +81,23 @@ public PseudoClass() {
     }
 
     /**
-     * There is only one PseudoClass instance for a given pseudoClass.
-     * @param pseudoClass the pseudo-class
-     * @return The PseudoClass for the given pseudoClass. Will not return null.
+     * Gets the {@code PseudoClass} instance for a given pseudo class name.
+     * <p>
+     * Note: There is only one {@code PseudoClass} instance for a given pseudo class name.
+     *
+     * @param pseudoClass the name of the pseudo class
+     * @return the {@code PseudoClass} instance for a given pseudo class name
+     * It will not return null.
```

Comment:
Since this is two sentences, you either need to restore the period after `name` or else add a semicolon there, and also remove the period after `null`. Either way, as long as you are add it, you might want to use code style for `null`.